portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path

```diff
@@ -76,6 +76,11 @@
 	<td>//div[contains(@class,'criteria-group')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>SEGMENT_CRITERIA_CARD</td>
+	<td>//div[contains(@class,"criteria-card-root")]</td>
+	<td></td>
+</tr>
```

Comment:
I think this path already exists

`ACSegments.assertCriteriaTable(criteriaName = "birthDate");`